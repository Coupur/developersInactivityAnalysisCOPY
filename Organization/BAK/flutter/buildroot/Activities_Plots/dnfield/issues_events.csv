id,date,event,creator_login
2480556103,2019-07-13 04:33:12,review_requested,dnfield
2480906024,2019-07-13 18:14:59,referenced,dnfield
2480906025,2019-07-13 18:14:59,merged,dnfield
2480906026,2019-07-13 18:15:00,closed,dnfield
2480906124,2019-07-13 18:15:14,head_ref_deleted,dnfield
2326026953,2019-05-08 03:08:01,review_requested,dnfield
2326026954,2019-05-08 03:08:01,review_requested,dnfield
2326033881,2019-05-08 03:14:22,review_requested,dnfield
2327801878,2019-05-08 16:33:25,referenced,dnfield
2327801883,2019-05-08 16:33:25,merged,dnfield
2327801891,2019-05-08 16:33:26,closed,dnfield
2327802127,2019-05-08 16:33:31,head_ref_deleted,dnfield
2191123158,2019-03-08 20:44:17,referenced,dnfield
2191123160,2019-03-08 20:44:17,merged,dnfield
2191123165,2019-03-08 20:44:17,closed,dnfield
2191123258,2019-03-08 20:44:20,head_ref_deleted,dnfield
2128772057,2019-02-09 15:10:54,referenced,dnfield
2128772058,2019-02-09 15:10:54,merged,dnfield
2128772059,2019-02-09 15:10:54,closed,dnfield
2098476118,2019-01-25 23:06:31,review_requested,dnfield
2098482037,2019-01-25 23:11:10,referenced,dnfield
2098482040,2019-01-25 23:11:10,merged,dnfield
2098482046,2019-01-25 23:11:10,closed,dnfield
2098482505,2019-01-25 23:11:36,head_ref_deleted,dnfield
2098409684,2019-01-25 22:21:22,review_requested,dnfield
2098419317,2019-01-25 22:27:28,closed,dnfield
2084360270,2019-01-19 05:05:54,referenced,dnfield
2084360271,2019-01-19 05:05:54,merged,dnfield
2084360272,2019-01-19 05:05:54,closed,dnfield
2084378417,2019-01-19 06:04:17,head_ref_deleted,dnfield
2041787754,2018-12-21 22:49:31,review_requested,dnfield
2041787755,2018-12-21 22:49:31,review_requested,dnfield
2041789485,2018-12-21 22:51:27,review_requested,dnfield
2041789487,2018-12-21 22:51:27,review_requested,dnfield
2041792313,2018-12-21 22:54:15,renamed,dnfield
2084052527,2019-01-18 22:08:57,referenced,dnfield
2084052529,2019-01-18 22:08:57,merged,dnfield
2084052530,2019-01-18 22:08:57,closed,dnfield
2084052626,2019-01-18 22:09:01,head_ref_deleted,dnfield
2036419997,2018-12-19 18:22:55,referenced,dnfield
2036420000,2018-12-19 18:22:55,merged,dnfield
2036420006,2018-12-19 18:22:56,closed,dnfield
2034500892,2018-12-19 01:06:08,referenced,dnfield
2034500893,2018-12-19 01:06:08,merged,dnfield
2034500897,2018-12-19 01:06:08,closed,dnfield
2034052618,2018-12-18 20:42:17,head_ref_force_pushed,dnfield
2034349330,2018-12-18 23:12:10,review_requested,dnfield
2034386979,2018-12-18 23:37:23,referenced,dnfield
2034386980,2018-12-18 23:37:23,merged,dnfield
2034386981,2018-12-18 23:37:23,closed,dnfield
2034387074,2018-12-18 23:37:26,head_ref_deleted,dnfield
2034499801,2018-12-19 01:05:29,referenced,dnfield
2034500926,2018-12-19 01:06:09,referenced,dnfield
2032289360,2018-12-18 08:19:50,review_requested,dnfield
2033688070,2018-12-18 17:57:48,referenced,dnfield
2033688073,2018-12-18 17:57:48,merged,dnfield
2033688097,2018-12-18 17:57:49,closed,dnfield
2033688239,2018-12-18 17:57:53,head_ref_deleted,dnfield
2033685251,2018-12-18 17:56:32,referenced,dnfield
2033685253,2018-12-18 17:56:32,merged,dnfield
2033685258,2018-12-18 17:56:32,closed,dnfield
