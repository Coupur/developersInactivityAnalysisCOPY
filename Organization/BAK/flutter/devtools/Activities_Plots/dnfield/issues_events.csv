id,date,event,creator_login
2267688993,2019-04-10 23:32:54,mentioned,dnfield
2267688994,2019-04-10 23:32:54,subscribed,dnfield
