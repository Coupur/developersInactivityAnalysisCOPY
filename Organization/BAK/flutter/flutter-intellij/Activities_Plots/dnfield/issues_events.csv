id,date,event,creator_login
2206768309,2019-03-15 15:16:55,mentioned,dnfield
2206768315,2019-03-15 15:16:55,subscribed,dnfield
