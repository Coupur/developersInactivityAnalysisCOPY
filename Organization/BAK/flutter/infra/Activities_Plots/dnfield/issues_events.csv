id,date,event,creator_login
2439077717,2019-06-25 22:06:50,review_requested,dnfield
2383337652,2019-06-02 19:00:22,review_requested,dnfield
2383426115,2019-06-02 22:40:30,referenced,dnfield
2383426116,2019-06-02 22:40:30,merged,dnfield
2383426117,2019-06-02 22:40:30,closed,dnfield
2344394511,2019-05-15 19:39:19,review_requested,dnfield
2344504127,2019-05-15 20:22:45,referenced,dnfield
2344504128,2019-05-15 20:22:45,merged,dnfield
2344504131,2019-05-15 20:22:45,closed,dnfield
2273612505,2019-04-12 21:28:16,referenced,dnfield
2273612507,2019-04-12 21:28:16,merged,dnfield
2273612509,2019-04-12 21:28:16,closed,dnfield
2273612584,2019-04-12 21:28:19,head_ref_deleted,dnfield
2247085887,2019-04-02 16:11:23,review_requested,dnfield
2247133186,2019-04-02 16:25:48,review_requested,dnfield
2247769520,2019-04-02 20:12:35,referenced,dnfield
2247769523,2019-04-02 20:12:35,merged,dnfield
2247769531,2019-04-02 20:12:36,closed,dnfield
2247769725,2019-04-02 20:12:40,head_ref_deleted,dnfield
2245280388,2019-04-02 04:45:57,review_requested,dnfield
2245280389,2019-04-02 04:45:57,review_requested,dnfield
2245280519,2019-04-02 04:46:03,referenced,dnfield
2245280520,2019-04-02 04:46:03,merged,dnfield
2245280522,2019-04-02 04:46:03,closed,dnfield
2245280585,2019-04-02 04:46:06,head_ref_deleted,dnfield
2247084507,2019-04-02 16:10:57,referenced,dnfield
2247769557,2019-04-02 20:12:36,referenced,dnfield
2244567637,2019-04-01 20:40:35,referenced,dnfield
2244567642,2019-04-01 20:40:35,merged,dnfield
2244567646,2019-04-01 20:40:35,closed,dnfield
2244567792,2019-04-01 20:40:38,head_ref_deleted,dnfield
2247084505,2019-04-02 16:10:57,referenced,dnfield
2247769554,2019-04-02 20:12:36,referenced,dnfield
2224368590,2019-03-23 14:15:06,review_requested,dnfield
2224368592,2019-03-23 14:15:06,review_requested,dnfield
2224424014,2019-03-23 16:22:18,referenced,dnfield
2224424015,2019-03-23 16:22:18,merged,dnfield
2224424017,2019-03-23 16:22:18,closed,dnfield
2224424066,2019-03-23 16:22:25,head_ref_deleted,dnfield
2223464940,2019-03-22 18:25:59,review_requested,dnfield
2223475311,2019-03-22 18:30:13,referenced,dnfield
2223475312,2019-03-22 18:30:13,merged,dnfield
2223475316,2019-03-22 18:30:13,closed,dnfield
2223475588,2019-03-22 18:30:19,head_ref_deleted,dnfield
2221644234,2019-03-22 04:38:05,review_requested,dnfield
2221656330,2019-03-22 04:50:54,referenced,dnfield
2221656331,2019-03-22 04:50:54,merged,dnfield
2221656334,2019-03-22 04:50:55,closed,dnfield
2223090409,2019-03-22 16:04:50,head_ref_deleted,dnfield
2221274910,2019-03-21 23:07:27,referenced,dnfield
2221274911,2019-03-21 23:07:27,merged,dnfield
2221274914,2019-03-21 23:07:27,closed,dnfield
2221305930,2019-03-21 23:29:32,head_ref_deleted,dnfield
2220958500,2019-03-21 20:31:04,referenced,dnfield
2220958503,2019-03-21 20:31:04,merged,dnfield
2220958506,2019-03-21 20:31:04,closed,dnfield
2220958670,2019-03-21 20:31:09,head_ref_deleted,dnfield
2220672821,2019-03-21 18:32:44,review_requested,dnfield
2220672823,2019-03-21 18:32:44,review_requested,dnfield
2220672825,2019-03-21 18:32:44,review_requested,dnfield
2220672831,2019-03-21 18:32:44,review_requested,dnfield
2221375877,2019-03-22 00:23:44,referenced,dnfield
2221375879,2019-03-22 00:23:44,merged,dnfield
2221375881,2019-03-22 00:23:44,closed,dnfield
2221375988,2019-03-22 00:23:48,head_ref_deleted,dnfield
2192175423,2019-03-10 07:37:01,review_requested,dnfield
2192175424,2019-03-10 07:37:01,review_requested,dnfield
2194518371,2019-03-11 16:32:11,referenced,dnfield
2194518372,2019-03-11 16:32:11,merged,dnfield
2194518376,2019-03-11 16:32:11,closed,dnfield
2194518726,2019-03-11 16:32:17,head_ref_deleted,dnfield
2159202492,2019-02-22 23:26:01,referenced,dnfield
2159202494,2019-02-22 23:26:01,merged,dnfield
2159202497,2019-02-22 23:26:01,closed,dnfield
2159202673,2019-02-22 23:26:10,head_ref_deleted,dnfield
2159079694,2019-02-22 22:06:59,mentioned,dnfield
2159079695,2019-02-22 22:07:00,subscribed,dnfield
2159102615,2019-02-22 22:18:56,referenced,dnfield
2159102616,2019-02-22 22:18:56,merged,dnfield
2159102619,2019-02-22 22:18:56,closed,dnfield
2143164629,2019-02-15 18:03:14,review_requested,dnfield
2143277665,2019-02-15 18:54:57,referenced,dnfield
2143277666,2019-02-15 18:54:57,merged,dnfield
2143277668,2019-02-15 18:54:57,closed,dnfield
2143277784,2019-02-15 18:55:01,head_ref_deleted,dnfield
2135278471,2019-02-12 23:44:46,review_requested,dnfield
2143907777,2019-02-16 01:18:28,referenced,dnfield
2143907779,2019-02-16 01:18:28,merged,dnfield
2143907782,2019-02-16 01:18:28,closed,dnfield
2143907820,2019-02-16 01:18:31,head_ref_deleted,dnfield
2126213601,2019-02-08 04:52:12,review_requested,dnfield
2126276579,2019-02-08 05:59:32,referenced,dnfield
2126276581,2019-02-08 05:59:32,merged,dnfield
2126276583,2019-02-08 05:59:32,closed,dnfield
2126276721,2019-02-08 05:59:42,head_ref_deleted,dnfield
2125862523,2019-02-07 23:39:30,review_requested,dnfield
2125862524,2019-02-07 23:39:30,review_requested,dnfield
2125871186,2019-02-07 23:45:09,referenced,dnfield
2125871187,2019-02-07 23:45:09,merged,dnfield
2125871191,2019-02-07 23:45:09,closed,dnfield
2125872030,2019-02-07 23:45:42,head_ref_deleted,dnfield
2125811345,2019-02-07 23:09:05,referenced,dnfield
2125811348,2019-02-07 23:09:05,merged,dnfield
2125811355,2019-02-07 23:09:06,closed,dnfield
2125811470,2019-02-07 23:09:09,head_ref_deleted,dnfield
2125740931,2019-02-07 22:29:45,review_requested,dnfield
2125740932,2019-02-07 22:29:45,review_requested,dnfield
2125742480,2019-02-07 22:30:26,referenced,dnfield
2125742482,2019-02-07 22:30:26,merged,dnfield
2125742489,2019-02-07 22:30:27,closed,dnfield
2125805199,2019-02-07 23:05:29,referenced,dnfield
2125807879,2019-02-07 23:07:03,head_ref_deleted,dnfield
2125811376,2019-02-07 23:09:06,referenced,dnfield
2125728489,2019-02-07 22:23:28,review_requested,dnfield
2125728490,2019-02-07 22:23:28,review_requested,dnfield
2125737281,2019-02-07 22:27:51,referenced,dnfield
2125737283,2019-02-07 22:27:51,merged,dnfield
2125737292,2019-02-07 22:27:51,closed,dnfield
2125737468,2019-02-07 22:27:56,head_ref_deleted,dnfield
2124849125,2019-02-07 18:12:03,review_requested,dnfield
2124873982,2019-02-07 18:22:06,referenced,dnfield
2124873983,2019-02-07 18:22:06,merged,dnfield
2124873994,2019-02-07 18:22:06,closed,dnfield
2124874192,2019-02-07 18:22:10,head_ref_deleted,dnfield
2121603174,2019-02-06 15:45:12,review_requested,dnfield
2121603175,2019-02-06 15:45:12,review_requested,dnfield
2121646029,2019-02-06 15:58:55,referenced,dnfield
2121646032,2019-02-06 15:58:56,merged,dnfield
2121646035,2019-02-06 15:58:56,closed,dnfield
2121646295,2019-02-06 15:59:02,head_ref_deleted,dnfield
2119304054,2019-02-05 18:32:59,review_requested,dnfield
2119304055,2019-02-05 18:33:00,review_requested,dnfield
2119356020,2019-02-05 18:54:14,referenced,dnfield
2119356022,2019-02-05 18:54:14,merged,dnfield
2119356023,2019-02-05 18:54:14,closed,dnfield
2116835108,2019-02-04 20:52:18,review_requested,dnfield
2116990795,2019-02-04 22:01:35,review_requested,dnfield
2117019505,2019-02-04 22:15:09,referenced,dnfield
2117019508,2019-02-04 22:15:09,merged,dnfield
2117019513,2019-02-04 22:15:09,closed,dnfield
2117019655,2019-02-04 22:15:13,head_ref_deleted,dnfield
2113885998,2019-02-02 15:41:41,review_requested,dnfield
2113945303,2019-02-02 18:00:08,referenced,dnfield
2113945304,2019-02-02 18:00:08,merged,dnfield
2113945307,2019-02-02 18:00:08,closed,dnfield
2105838069,2019-01-30 08:29:15,review_requested,dnfield
2105838205,2019-01-30 08:29:19,review_requested,dnfield
2105839488,2019-01-30 08:29:50,review_requested,dnfield
2105840158,2019-01-30 08:30:08,renamed,dnfield
2107149780,2019-01-30 17:06:28,referenced,dnfield
2107149782,2019-01-30 17:06:28,merged,dnfield
2107149794,2019-01-30 17:06:28,closed,dnfield
2107149992,2019-01-30 17:06:32,head_ref_deleted,dnfield
2105617922,2019-01-30 06:09:12,review_requested,dnfield
2105621047,2019-01-30 06:11:53,referenced,dnfield
2105621049,2019-01-30 06:11:53,merged,dnfield
2105621050,2019-01-30 06:11:53,closed,dnfield
2069955839,2019-01-12 05:26:11,referenced,dnfield
2069955840,2019-01-12 05:26:11,merged,dnfield
2069955841,2019-01-12 05:26:11,closed,dnfield
2069863960,2019-01-12 01:06:34,review_requested,dnfield
2069865173,2019-01-12 01:08:29,referenced,dnfield
2069865174,2019-01-12 01:08:30,merged,dnfield
2069865177,2019-01-12 01:08:30,closed,dnfield
