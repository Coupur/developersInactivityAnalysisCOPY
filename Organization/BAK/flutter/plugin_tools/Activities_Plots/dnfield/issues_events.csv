id,date,event,creator_login
1915569340,2018-10-19 19:32:42,review_requested,dnfield
1915449727,2018-10-19 18:29:28,review_requested,dnfield
1915565211,2018-10-19 19:30:09,referenced,dnfield
1915565213,2018-10-19 19:30:09,merged,dnfield
1915565214,2018-10-19 19:30:09,closed,dnfield
1913579821,2018-10-18 23:25:04,review_requested,dnfield
1913662696,2018-10-19 00:45:04,referenced,dnfield
1913662697,2018-10-19 00:45:04,merged,dnfield
1913662698,2018-10-19 00:45:04,closed,dnfield
1913532044,2018-10-18 22:46:30,review_requested,dnfield
1913563728,2018-10-18 23:11:19,referenced,dnfield
1913563729,2018-10-18 23:11:19,merged,dnfield
1913563731,2018-10-18 23:11:19,closed,dnfield
1913535309,2018-10-18 22:49:07,referenced,dnfield
1913535311,2018-10-18 22:49:07,merged,dnfield
1913535315,2018-10-18 22:49:07,closed,dnfield
