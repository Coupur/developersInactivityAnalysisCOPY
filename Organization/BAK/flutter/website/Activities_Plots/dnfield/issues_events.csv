id,date,event,creator_login
2477740141,2019-07-12 02:28:40,mentioned,dnfield
2477740142,2019-07-12 02:28:40,subscribed,dnfield
2474061683,2019-07-10 22:22:44,mentioned,dnfield
2474061684,2019-07-10 22:22:44,subscribed,dnfield
2474094282,2019-07-10 22:41:58,mentioned,dnfield
2474094283,2019-07-10 22:41:58,subscribed,dnfield
2477355136,2019-07-11 21:47:21,mentioned,dnfield
2477355138,2019-07-11 21:47:21,subscribed,dnfield
2477503599,2019-07-11 23:14:58,mentioned,dnfield
2477503600,2019-07-11 23:14:59,subscribed,dnfield
2143980848,2019-02-16 03:49:28,mentioned,dnfield
2143980849,2019-02-16 03:49:28,subscribed,dnfield
2032826658,2018-12-18 12:21:38,mentioned,dnfield
2032826660,2018-12-18 12:21:38,subscribed,dnfield
1973435503,2018-11-18 22:55:39,review_requested,dnfield
1973486316,2018-11-19 00:44:15,review_requested,dnfield
1975118447,2018-11-19 16:27:46,review_requested,dnfield
1932939105,2018-10-29 17:31:38,mentioned,dnfield
1932939106,2018-10-29 17:31:38,subscribed,dnfield
1932990460,2018-10-29 17:50:52,mentioned,dnfield
1932990461,2018-10-29 17:50:52,subscribed,dnfield
