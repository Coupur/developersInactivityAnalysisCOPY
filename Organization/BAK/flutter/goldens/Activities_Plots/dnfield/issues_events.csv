id,date,event,creator_login
2298484860,2019-04-24 20:59:19,referenced,dnfield
2298484861,2019-04-24 20:59:19,merged,dnfield
2298484864,2019-04-24 20:59:19,closed,dnfield
2298484979,2019-04-24 20:59:22,head_ref_deleted,dnfield
2298483425,2019-04-24 20:58:43,referenced,dnfield
2298484886,2019-04-24 20:59:20,referenced,dnfield
2064314169,2019-01-09 21:56:00,referenced,dnfield
2064314172,2019-01-09 21:56:00,merged,dnfield
2064314180,2019-01-09 21:56:00,closed,dnfield
